<map id="sysio::chain::controller_impl::scheduled_failure_is_subjective" name="sysio::chain::controller_impl::scheduled_failure_is_subjective">
<area shape="rect" id="Node000001" title=" " alt="" coords="5,5,167,61"/>
<area shape="rect" id="Node000002" href="$classfc_1_1exception.html#a79a36225cde10dcd23057bbe19d3390f" title=" " alt="" coords="441,20,573,46"/>
<area shape="poly" id="edge1_Node000001_Node000002" title=" " alt="" coords="167,28,215,27,393,27,426,27,425,33,393,32,215,32,167,33"/>
<area shape="rect" id="Node000003" href="$structsysio_1_1chain_1_1controller__impl.html#ad0d5a921dd01458d1464f63b0f0121c6" title=" " alt="" coords="215,41,393,82"/>
<area shape="poly" id="edge2_Node000001_Node000003" title=" " alt="" coords="167,41,199,45,199,50,166,46"/>
<area shape="poly" id="edge3_Node000003_Node000002" title=" " alt="" coords="393,46,426,42,426,47,394,52"/>
</map>
